.devcontainer/Dockerfile

```diff
@@ -15,3 +15,5 @@ RUN groupadd -g 1000 dev \
  && chown -R dev:dev /home/dev
 
 USER dev
+
+ENV DOTNET_CLI_TELEMETRY_OPTOUT=1
```

Comment:
Why this and the other docker changes?

Reply:
Oops. I'll move them to a separate PR. The gist is that the latest C# extension has broken intellisense, and I don't like being spied on with telemetry. 😜